libvast/src/system/node.cpp

```diff
@@ -196,7 +197,7 @@ status_command(const command::invocation&, caf::actor_system& sys) {
 }
 
 maybe_actor spawn_accountant(node_actor* self, spawn_arguments&) {
-  auto accountant = self->spawn<monitored>(system::accountant);
+  auto accountant = self->spawn<linked>(system::accountant);
```

Comment:
I thought about this more. It's always possible that an actor receives multiple EXIT messages. What's important is that the processing is idempotent, i.e., the shutdown process is initiated only once. In the `shutdown` helper function, I've implemented this by resetting the EXIT handler so that it only performs a hard kill when another EXIT message with reason `kill` arrives.

Reply:
I agree that it shouldnt matter that we get multiple EXIT messages, but in this case linking also implies that if the accountant goes down, the node will go down. Do we want that behaviour?

Reply:
I think we do. The accountant is a low-complexity piece and should be robust. If it fails, we have a bigger problem that we probably shouldn't ignore. Therefore, I suggest fate sharing.